app/views/docs/getting-started-for-flutter.phtml

```diff
@@ -33,21 +33,18 @@ $version = (isset($versions['flutter'])) ? $versions['flutter'] : '';
 
 <p>For <b>Android</b> first add your app <u>name</u> and <u>package name</u>, Your package name is generally the <b>applicationId</b> in your app-level <a href="https://github.com/appwrite/playground-for-flutter/blob/0fdbdff98384fff940ed0b1e08cf14cfe3a2be3e/android/app/build.gradle#L41" target="_blank" rel="noopener">build.gradle</a> file. By registering your new app platform, you are allowing your app to communicate with the Appwrite API.</p>
 
-<p>In order to capture the Appwrite OAuth callback url, the following activity needs to be added to your <a href="https://github.com/appwrite/playground-for-flutter/blob/master/android/app/src/main/AndroidManifest.xml" target="_blank" rel="noopener">AndroidManifest.xml</a>. Be sure to relpace the <b>[PROJECT_ID]</b> string with your actual Appwrite project ID. You can find your Appwrite project ID in you project settings screen in your Appwrite console.</p>
+<p>In order to capture the Appwrite OAuth callback url, the following activity needs to be added below `application` tag along side existing `activity` tags to your <a href="https://github.com/appwrite/playground-for-flutter/blob/master/android/app/src/main/AndroidManifest.xml" target="_blank" rel="noopener">AndroidManifest.xml</a>. Be sure to relpace the <b>[PROJECT_ID]</b> string with your actual Appwrite project ID. You can find your Appwrite project ID in you project settings screen in your Appwrite console.</p>
 
 <div class="ide" data-lang="html" data-lang-label="XML">
-    <pre class="line-numbers"><code class="prism language-xml" data-prism><?php echo $this->escape('<manifest>
-  <application>
-    <activity android:name="com.linusu.flutter_web_auth.CallbackActivity" >
-      <intent-filter android:label="flutter_web_auth">
-        <action android:name="android.intent.action.VIEW" />
-        <category android:name="android.intent.category.DEFAULT" />
-        <category android:name="android.intent.category.BROWSABLE" />
-        <data android:scheme="appwrite-callback-[PROJECT_ID]" />
-      </intent-filter>
-    </activity>
-  </application>
-</manifest>
+    <pre class="line-numbers"><code class="prism language-xml" data-prism><?php echo $this->escape('<!-- Add this between `application` tag along side existing `activity` tags -->
+<activity android:name="com.linusu.flutter_web_auth.CallbackActivity" >
```

Comment:
How about writing Like this?
```xml
<manifest ...... >
.
.
<application ................ >
.
.
.
<!-- Add inside the existing application tag along the side of existing activity tags  ---->
<activity android:name="com.linusu.flutter_web_auth.CallbackActivity" >
<intent-filter android:label="flutter_web_auth">
    <action android:name="android.intent.action.VIEW" />
    <category android:name="android.intent.category.DEFAULT" />
    <category android:name="android.intent.category.BROWSABLE" />
    <data android:scheme="appwrite-callback-[PROJECT_ID]" />
  </intent-filter>
</activity>
.
.
.
.
</application>
.
.
.
</manifest>
```
That's how I was thinking . What do you guyz think?

Reply:
Maybe with `...` instead of multiple `.` with line breaks

Reply:
Yeah just something like that.  Couldn't express properly myself though😅

Reply:
@2002Bishwajeet Updated, do check.